package.json

```diff
@@ -5,27 +5,24 @@
     "type": "git",
     "url": "https://github.com/villander/ember-engines-router-service"
   },
-  "workspaces": [
-    "ember-engines-router-service",
-    "test-app"
-  ],
   "scripts": {
-    "build": "yarn workspace ember-engines-router-service run build",
-    "lint": "yarn workspaces run lint",
-    "lint:fix": "yarn workspaces run lint:fix",
-    "prepare": "yarn build",
+    "build": "pnpm -F ember-engines-router-service build",
+    "lint": "pnpm -r lint --if-present",
+    "lint:fix": "pnpm -r lint:fix --if-present",
+    "prepare": "pnpm build",
     "release": "release-it",
-    "start": "concurrently 'yarn:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
-    "start:addon": "yarn workspace ember-engines-router-service run start",
-    "start:test-app": "yarn workspace test-app run start",
-    "test": "yarn workspaces run test",
-    "test:ember": "yarn workspace test-app run test:ember"
+    "start": "concurrently 'pnpm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "pnpm -F ember-engines-router-service start",
+    "start:test-app": "pnpm -F test-app start",
+    "test": "pnpm -r test --if-present",
+    "test:ember": "pnpm -F test-app test:ember"
   },
   "devDependencies": {
     "@release-it-plugins/lerna-changelog": "^7.0.0",
     "@release-it-plugins/workspaces": "^4.2.0",
     "release-it": "^17.5.0"
   },
+  "packageManager": "pnpm@10.12.1+sha512.f0dda8580f0ee9481c5c79a1d927b9164f2c478e90992ad268bbb2465a736984391d6333d2c327913578b2804af33474ca554ba29c04a8b13060a717675ae3ac",
```

Comment:
Why is it needed? It does not seem to be the right thing to do with this long version. Why can't we simply use `10.12.1`?  

https://stackoverflow.com/questions/71747609/how-to-specify-packagemanager-in-package-json

Reply:
corepack adds a sha for integrity check